{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a",
                            "versionType": "custom"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "FTPServer.py in pyftpdlib before 0.2.0 allows remote attackers to cause a denial of service via a long command."
                }
            ],
            "problemtypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2021-12-15",
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre"
            },
            "references": [
                {
                    "name": "http://code.google.com/p/pyftpdlib/source/diff?spec=svn20&r=20&format=side&path=/trunk/pyftpdlib/FTPServer.py",
                    "url": "http://code.google.com/p/pyftpdlib/source/diff?spec=svn20&r=20&format=side&path=/trunk/pyftpdlib/FTPServer.py"
                },
                {
                    "name": "http://code.google.com/p/pyftpdlib/issues/detail?id=3",
                    "url": "http://code.google.com/p/pyftpdlib/issues/detail?id=3"
                },
                {
                    "name": "http://code.google.com/p/pyftpdlib/source/detail?r=20",
                    "url": "http://code.google.com/p/pyftpdlib/source/detail?r=20"
                },
                {
                    "name": "http://code.google.com/p/pyftpdlib/source/browse/trunk/HISTORY",
                    "url": "http://code.google.com/p/pyftpdlib/source/browse/trunk/HISTORY"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2007-6739",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "FTPServer.py in pyftpdlib before 0.2.0 allows remote attackers to cause a denial of service via a long command."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "http://code.google.com/p/pyftpdlib/source/diff?spec=svn20&r=20&format=side&path=/trunk/pyftpdlib/FTPServer.py",
                            "refsource": "CONFIRM",
                            "url": "http://code.google.com/p/pyftpdlib/source/diff?spec=svn20&r=20&format=side&path=/trunk/pyftpdlib/FTPServer.py"
                        },
                        {
                            "name": "http://code.google.com/p/pyftpdlib/issues/detail?id=3",
                            "refsource": "CONFIRM",
                            "url": "http://code.google.com/p/pyftpdlib/issues/detail?id=3"
                        },
                        {
                            "name": "http://code.google.com/p/pyftpdlib/source/detail?r=20",
                            "refsource": "CONFIRM",
                            "url": "http://code.google.com/p/pyftpdlib/source/detail?r=20"
                        },
                        {
                            "name": "http://code.google.com/p/pyftpdlib/source/browse/trunk/HISTORY",
                            "refsource": "CONFIRM",
                            "url": "http://code.google.com/p/pyftpdlib/source/browse/trunk/HISTORY"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "cveId": "CVE-2007-6739",
        "dateUpdated": "2021-12-15",
        "state": "PUBLIC"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}